CHANGELOG.md

```diff
@@ -10,9 +10,11 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## Added
```

Comment:
Totally unrelated and not blocking, but if we're fixing the missing newline below, can we make this an `h3` header instead of `h2`?

Reply:
yep, I'll fix this and also a typo in one of the items in the v0.18.0 line items/.